resources/assets/js/components/Application/BasicInfo/BasicInfoPage.tsx

```diff
@@ -55,6 +61,9 @@ const BasicInfoPage: React.FunctionComponent<BasicInfoPageProps> = ({
     values: ApplicationNormalized,
   ): Promise<void> => {
     await updateApplication(values);
+    await dispatch(
+      updateApplicationStep(applicationId, ApplicationStepId.experience),
+    );
```

Comment:
I think it's better to update the state here, because if we reach the next step before the request is done then you'll get a flash  of the error status on the ProgressBar. We could avoid this by checking if the steps are loading and adding a new `loading` state, with the loading icon. However, I think it's cleaner to leave it like this.